### ISSUE TITLE
IIS 10 deploy Rdotnet Issue

### ISSUE THREAD

**Issue by manjotsingh20** [jmp75/rdotnet#28]
Hi All,

I am facing deployment issues in IIS 10
**Error in library(RODBC) : there is no package called 'RODBC'** same with other packages also.
Tried changing path also but nothing works for me
But in VS2015 its working fine

Thanks

**Comment by manjotsingh20** [comment]
@skyguy94 can you help me to solve this issue because i tried all possible ways but no luck

**Comment by evolvedmicrobe** [comment]
I would verify that the R you are using is the same (check for 32 bit/ 64 bit conflicts), also evaluate the returned values from R.home() as well as the .libPaths()  in both VS2011 and IS10 to check for differences.

**Comment by manjotsingh20** [comment]
Yes i tried all ways but no luck tried changing environment variable path accordingly but nothing works

**Comment by skyguy94** [comment]
@manjotsingh20 This is the very common R.Net Library load issue when running from IIS.

Please follow the instructions in this issue (https://rdotnet.codeplex.com/workitem/117).

**Comment by manjotsingh20** [comment]
@skyguy94 I tried adding C:\Program Files\R\R-3.2.3\bin\i386 and for 64 bit, another way also that's mentioned in that url also but no luck

**Comment by skyguy94** [comment]
Did you try the sample mentioned in that work item, ie: `print(summary((1:5))`? Does that work?

If that doesn't work for you, I don't know what to tell you without some unguided troubleshooting on your end. The PATH issue is a well documented and well understood compatibility problem between IIS and R.

**Comment by manjotsingh20** [comment]
okay will troubleshoot this..
